index.html

```diff
@@ -74,20 +77,458 @@ <h4 class="hero__subheading">
+                      />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="socials__link">
+                      <img
+                        class="link__img"
+                        src="./assets/images/icons/twitter.png"
+                        alt="twitter"
+                      />
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <div class="flex__item">
+              <img
+                class="team__photo"
+                src="./assets/images/team_Avie.png"
+                alt="photo Avie"
+              />
+              <div class="team__info">
+                <h5 class="team__name">Avie Beaton</h5>
+                <h6 class="team__position">CO Founder</h6>
+                <ul class="team__socials">
+                  <li>
+                    <a href="#" class="socials__link">
+                      <img
+                        class="link__img"
+                        src="./assets/images/icons/facebook.png"
+                        alt="facebook"
+                      />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="socials__link">
+                      <img
+                        class="link__img"
+                        src="./assets/images/icons/inst.png"
+                        alt="inst"
+                      />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="socials__link">
+                      <img
+                        class="link__img"
+                        src="./assets/images/icons/twitter.png"
+                        alt="twitter"
+                      />
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
       </section>
-      <section>
-        <div class="container">Sec6</div>
+
+      <section class="contact">
+        <div class="container">
+          <img
+            class="contact__image"
+            src="./assets/images/Contact.png"
+            alt="contact background"
+          />
+          <div class="contact__main">
+            <h5 class="contact__heading">Contact Us</h5>
+            <h2 class="contact__subheading">Make an Appointment</h2>
+            <form class="contact__form" action="#" method="post">
+              <input
+                class="form__appointment"
+                type="text"
+                id="fullname"
+                name="fullname"
+                placeholder="Full Name"
+                required
+              />
+              <input
+                class="form__appointment"
+                type="email"
+                placeholder="example@gmail.com"
+                required
+              />
+              <select class="form__appointment appointment--select">
```

Comment:
rethink the class naming. "form__appointment" tells me that this is class for an appointment form. But its class of any form element, which sound confusing.

maybe "form__element form__element--select" is better and more generic?